setup.py

```diff
@@ -6,7 +7,12 @@
 __url__ = 'https://github.com/mahmoud/glom'
 __license__ = 'BSD'
 
-with open('README.md') as read_me:
+
+open_kwarg = {}
+if sys.version_info[0] == 3:
+    open_kwarg['encoding'] = 'utf-8'
+
+with open('README.md', **open_kwarg) as read_me:
```

Comment:
huh, surprisingly I haven't had to make this change in other py23 setup.pys. E.g., https://github.com/SimpleLegal/pocket_protector/blob/4154732b3891132c8f525ae2005b778ea49ae232/setup.py#L16